deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml

```diff
@@ -188,6 +190,8 @@ spec:
             - mountPath: /etc/storage_ibmc/cluster_info
               name: cluster-info
```

Comment:
Even, cluster info mount and customer-auth can be removed, because using the CI changed sidecar image, it does not need to be mounted. Please remove the same and test once.